docs/user-guide/persistent-volumes/walkthrough.md

```diff
@@ -36,6 +36,31 @@ NAME      LABELS       CAPACITY      ACCESSMODES   STATUS      CLAIM     REASON
 pv0001    type=local   10737418240   RWO           Available 
 ```
 
+### Access Control
+Storage configured with GID will only allow writing by pods using the same GID.
+Mismatched or missing GIDs will cause `permission denied` errors. Adding a GID
+to a `PersistentVolume` via the `pv.beta.kubernetes.io/gid` annotation allows
+`kubelet` to automatically add the GID to the pod that requires it. No
+coordination between an admin and end user is required. Setting the GID
+automatically is safe because this is only done for `PersistentVolumes` which
+are accessed through a `PersistentVolumeClaim`.
+
+To add the volume's GID to the volume you use the `pv.beta.kubernetes.io/gid`
+annotation as follows:
+
+```yaml
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: pv1
+  annotations:
+    pv.beta.kubernetes.io/gid: "1234"
+#...
+```
+
```

Comment:
@wongma7 Add an explanation of what this means for the pod.  Make sure to note that currently, the GIDs associated with PVs a pod consumes will not be present on the pod resource itself.  This reminds me, I need to dig up the issue for security context status and update it with this case.